File: Selenium/Activities/Activity1.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args){
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net");
        String title=driver.getTitle();
        System.out.println("Title of webpage:"+title);
        WebElement aboutUsBtn=driver.findElement(By.id("about-link"));
        aboutUsBtn.click();
        System.out.println("Title of the webpage now:"+driver.getTitle());
        driver.close();
    }
}

File: Selenium/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Navigate to the URL
    driver.get("https://v1.training-support.net/")
    print("Title of the webpage:" + driver.title)
    driver.find_element(By.ID, "about-link").click()
    print("Title of the webpage now:" + driver.title)
    driver.close()
